core/src/main/java/tachyon/worker/WorkerStorage.java

```diff
@@ -611,6 +614,23 @@ private long getUsedBytes() {
     }
     return usedBytes;
   }
+  
+  /**
+   * Get used bytes of each storage level(alias)
+   * 
+   * @return used bytes of each storage level(alias)
+   */
+  private List<Long> getUsedBytesByAlias() {
+    List<Long> usedBytes = new ArrayList<Long>();
+    for (int i = 0; i < StorageLevelAlias.values().length; i++) {
```

Comment:
I write getUsedBytesByAlias() in this way to resolve the different setup of workers. We just need to care about if the storageTier is MEM or SSD or HDD, and don't care about the order of storageTiers. And in this way, we don't need to send storageTier information , only need to send how many bytes are used because it is already ordered.